refinery/tool_manager/models.py

```diff
@@ -509,8 +508,7 @@ class Meta:
         )
 
     def get_container_input_dict(self):
-        """
-        Create a dictionary containing information that Dockerized
+        """Create a dictionary containing information that Dockerized
```

Comment:
Why the format change?

Reply:
Unnecessary newline (https://www.python.org/dev/peps/pep-0257/#id17)

Reply:
It'd be great to automate any rules we want to enforce. I personally find the left easier to read.